src/components/ChatEntry.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const senderClass =
+    props.sender === 'Estragon' ? 'chat-entry remote' : 'chat-entry local';
+
+  const heart = props.liked ? '❤️' : '🤍';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div key={props.id} className={senderClass}>
```

Comment:
There's no need for a `key` here. We only put a key on a component at the spot where it's being rendering into a list of components, not in the definition of the component structure itself.

Reply:
The only part of the class that changes based on being the local or remote sender is `local` or `remote`. `chat-entry` doesn't change. So consider only storing `local` or `remote` in `senderClass `, then modify this to
```js
    <div className={`chat-entry ${senderClass}`}>
```

This can help us avoid needing to build a huge combination of class name strings if we had other classes to toggle between.